lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,19 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.last.nil?
+            ""
+          else
+            "#{comments.last.body.inspect}"
+          end
+        end
+
+        def approved?
```

Comment:
Checking the `minimum_approvals` for `nil` is the same as checking that `.to_i == 0` as `nil.to_i == 0`, so let's at least update that.